src/Impostor.Server/Net/State/Game.Outgoing.cs

```diff
@@ -142,5 +142,14 @@ private ValueTask SendObjectSpawn(InnerNetObject obj, int? targetClientId = null
             writer.EndMessage();
             return FinishGameDataAsync(writer, targetClientId);
         }
+
+        private ValueTask SendObjectDespawn(InnerNetObject obj, int? targetClientId = null)
```

Comment:
```suggestion
        private ValueTask SendObjectDespawnAsync(InnerNetObject obj, int? targetClientId = null)
```
maybe `DespawnAsync`?